frontend/src/components/RequestFormInputs.vue

```diff
@@ -228,8 +224,18 @@ const executeFaucetRequest = async () => {
 };
 const { active: faucetRequestActive, run: runFaucetRequest } =
   createAsyncProcess(executeFaucetRequest);
+
+const gasFeesTooltipText = `The fee amount is composed of three parts:
+  - the gas reimbursement fee
+  - the liquidity provider fee
+  - the Beamer service fee
```

Comment:
Can we replace these with bullet points? I think it would look nicer?

Reply:
yes we can